### PR TITLE
Allow html tags for formatting text

### PR DESCRIPTION
**Pull request by davidsbailey**
Allow html tags for formatting text, and suppress warnings when stripping harmless attributes.

The impetus for this was a user request for being able to print table data in some form of structured output, plus a quick pass through [new relic insights](https://insights.newrelic.com/accounts/501463/query?hello=overview&query=SELECT%20currentUrl,channelId,removedHtml,safeHtml,unsafeHtml,sanitizationTarget,userAgentName,userAgentOS,userAgentVersion,city,regionCode,countryCode%20from%20PageAction%20where%20actionName%20%3D%20%27SanitizedLevelHtml%27%20since%206%20weeks%20ago%20limit%201000) for other legitimate HTML tags / attributes which people are trying to use.

screen shot 2016-04-04 at 2 54 58 pm

### CODE REVIEW

File: apps/src/applab/sanitizeHtml.js [hunk 1]
```diff
@@ -47,10 +47,29 @@ function warnAboutUnsafeHtml(warn, unsafe, safe, warnings) {
     return 0;
   };
 
+  // Do not warn when these attributes are removed.
+  var ignoredAttributes = ['kl_vkbd_parsed'];
```
Comment (Bjvanminnen) [inline apps/src/applab/sanitizeHtml.js]: What is `kl_vkbd_parsed`?

Comment (davidsbailey) [inline apps/src/applab/sanitizeHtml.js]: I'm not really sure, but @islemaster  and I had previously hypothesized that they are from some browser plugin. See previous discussion here: https://github.com/code-dot-org/code-dot-org/pull/6895#issuecomment-186941112

Comment (Bjvanminnen) [inline apps/src/applab/sanitizeHtml.js]: Should `pmbx_context` also be in this list?

Might also be worth a short comment in this file explaining that this is some mysterious attribute that we dont know much about, but know that we're fine with removing it silently.

**Comment by Bjvanminnen** [comment]
lgtm

File: apps/src/applab/sanitizeHtml.js (hunk 1 above)
Comment (davidsbailey) [inline apps/src/applab/sanitizeHtml.js]: Done.

**Pull request merged**